test/fixtures/basic/ignore-joiners.html

```diff
@@ -0,0 +1,29 @@
+<div class="basic-ignore-joiners">
+    <div>
+        <p>
+            Lo&shy;rem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+            diam nonumy eirmod tempor invidunt ut labore et dolore magna
+            aliquyam erat, sed diam voluptua. At vero eos et accusam et justo
+            duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata
+            sanctus est lorem ipsum dolor sit. Lo&#8203;rem ip&#8204;sum dolor
+            sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor
+            invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+            At vero eos et accusam et justo duo dolores et ea rebum. Stet clita
+            kasd gubergren, no sea takimata sanctus est lorem ips&#8205;um dolor
+            sit amet.
+        </p>
+    </div>
+    <div>
+        <p>
```

Comment:
typo: indentation
Also occurs in other HTML files. What editor are you using? In Atom you could just run Strg + Alt + B when using atom-beautify.